shapely/README.md

```diff
@@ -0,0 +1,74 @@
+# Report for Assignment 1
+
+## Project chosen
+
+Name: Shapely
+
+URL: https://github.com/shapely/shapely
+
+Number of lines of code and the tool used to count it: 25027 tokei
+
+Programming language: Python
+
+## Coverage measurement
+
+### Existing tool - coverage.py
+
+The coverage tool executed on the project is [coverage.py](https://coverage.readthedocs.io/en/latest/index.html).
+
+The tool can be executed through the following steps (note that you need Python 3.x installed):
+
+1. **Clone the repository**
+```bash
+git clone https://github.com/Papajo-boban/shapely.git
+cd shapely
+```
+2. **Create a virtual environment**
+```bash
+python3 -m venv .venv
+```
+3. **Activate virtual environment**
+```bash
+source .venv/bin/activate # for Unix/macOS
+.venv\Scripts\activate.bat # for windows
+```
+4. **Install project and test dependencies**
+```bash
+pip install -e .
+pip install pytest coverage matplotlib # matplotlib was needed as some tests were automatically skipped without it
+```
+5. **Run the tests using coverage.py**
+```bash
+coverage run -m pytest
+```
+6. **Create html coverage information and open it in your browser**
+```bash
+coverage html
+```
+7. **Result**
+![alt text](<Screenshot 2024-06-22 142207.png>)

```

Comment:
I think this part is good as it was in the main README, we don't have to do this step for each individual branch. Can you revert it to the original?